### DISCUSSION TITLE
Trouble downloading WixSharp.. What am I missing?

### DISCUSSION THREAD

**Discussion by MarkHaroldson** [oleg-shilo/wixsharp#1442] [General]
Hi, I am trying to create a sample ".msi" package in c#.

When I download any of the WixSharp packages via NUGET I only get msbuild targets and not any binaries that I can use in my C# code.

using WixSharp; // <== This does not resolve.

What am I missing?

Thanks in advance.

**Comment by MarkHaroldson** [comment]
I figured it out. I added the the nuget package to a .Net Core project, rather than a .Net Framework project.

**Comment by oleg-shilo** [comment]
Happens to me too, from time to time. :)

I wish nuget manager notifies about this explicitly.
